src/T2DMap.cpp

```diff
@@ -2194,6 +2191,10 @@ void T2DMap::paintAreaExits(QPainter& painter, QPen& pen, QList<int>& exitList,
 // Work out text for information box, need to offset if room selection widget is present
 void T2DMap::paintMapInfo(const QElapsedTimer& renderTimer, QPainter& painter, const bool showingCurrentArea, QColor& infoColor)
 {
+    if (!mpMap->mpMapper->mMapInfoContributorManager->contributors.keys().toSet().intersects(mpHost->mMapInfoProviders)) {
```

Comment:
See the `QT_VERSION_CHECK` we've had to use elsewhere with `.toSet()` because of Qt changes to remove the warning